.github/workflows/index-resources.yml

```diff
@@ -45,7 +45,9 @@ jobs:
           HEROKU_TOKEN: ${{ secrets.HEROKU_TOKEN_READ_PROTECTED }}
           IS_WORKFLOW_CALL: ${{ inputs.is-workflow-call || false }}
         run: |
-          echo "ref-matrix=$(./scripts/get-resource-index-ref-matrix)" | tee -a "$GITHUB_OUTPUT"
+          # Assign to variable before output to ensure error exit code propagates to GH Action job
+          ref_matrix=$(./scripts/get-resource-index-ref-matrix)
+          echo "ref_matrix=$ref_matrix" | tee -a "$GITHUB_OUTPUT"
```

Comment:
Sigh, I'm just have a bad week with typos. Fixed in https://github.com/nextstrain/nextstrain.org/commit/9d26e68cb5dd13de1924d9185eec6142d4428917